contracts/reverseRegistrar/IL2ReverseRegistrar.sol

```diff
@@ -0,0 +1,55 @@
+pragma solidity >=0.8.4;
+
+interface IL2ReverseRegistrar {
+    function setName(string memory name) external returns (bytes32);
```

Comment:
Why do we need to return `bytes32` for transactions? `setName` on L1 doesn't return anything https://github.com/ensdomains/ens-contracts/blob/staging/contracts/resolvers/profiles/NameResolver.sol#L15 . The same applies to all `set*` functions.

Reply:
This point still stands

Reply:
The original reverseRegistrar does this: https://github.com/ensdomains/ens-contracts/blob/staging/contracts/reverseRegistrar/IReverseRegistrar.sol

Reply:
Isn't it the contract you added several months ago? What was the intention at that time? If it's wrong then it should be fixed now unless it requires some sort of complex migration work as the change itself is not so critical